tss-esapi/tests/integration_tests/error_tests/return_code_tests/base_tests.rs

```diff
@@ -168,70 +168,108 @@ fn test_invalid_conversions() {
+        BaseError::NotImplemented
+    );
+    test_display_trait_impl!("A context structure is bad.", BaseError::BadContext);
+    test_display_trait_impl!(
+        "Passed in ABI version doesn't match called module's ABI version.",
+        BaseError::AbiMismatch
+    );
+    test_display_trait_impl!(
+        "A pointer is NULL that isn't allowed to be NULL.",
+        BaseError::BadReference
+    );
+    test_display_trait_impl!(
+        "A buffer isn't large enough.",
+        BaseError::InsufficientBuffer
+    );
+    test_display_trait_impl!(
+        "Function called in the wrong order.",
+        BaseError::BadSequence
+    );
+    test_display_trait_impl!(
+        "Fails to connect to next lower layer.",
+        BaseError::NoConnection
+    );
+    test_display_trait_impl!(
+        "Operation timed out; function must be called again to be completed.",
+        BaseError::TryAgain
+    );
+    test_display_trait_impl!("IO failure.", BaseError::IoError);
+    test_display_trait_impl!("A parameter has a bad value.", BaseError::BadValue);
+    test_display_trait_impl!("Operation not permitted.", BaseError::NotPermitted);
+    test_display_trait_impl!(
+        "The TPM command doesn't use the number of sessions provided by the caller.",
+        BaseError::InvalidSessions
+    );
+    test_display_trait_impl!("A session with decrypt set in its SessionAttributes(TPMA_SESSION_DECRYPT bit set) was passed to a TPM command that doesn't support encryption of the first command parameter.", BaseError::NoDecryptParam);
+    test_display_trait_impl!("A session with encrypt set in its SessionAttributes(TPMA_SESSION_ENCRYPT bit set) was passed to a TPM command that doesn't support encryption of the first response parameter.", BaseError::NoEncryptParam);
+    test_display_trait_impl!("Size of a parameter is incorrect.", BaseError::BadSize);
+    test_display_trait_impl!("Response is malformed.", BaseError::MalformedResponse);
+    test_display_trait_impl!("Context not large enough.", BaseError::InsufficientContext);
+    test_display_trait_impl!(
+        "Response is not long enough.",
+        BaseError::InsufficientResponse
+    );
+    test_display_trait_impl!(
+        "Unknown or unusable TCTI version.",
+        BaseError::IncompatibleTcti
+    );
+    test_display_trait_impl!("Functionality not supported.", BaseError::NotSupported);
+    test_display_trait_impl!("TCTI context is bad.", BaseError::BadTctiStructure);
+    test_display_trait_impl!("Memory allocation failed.", BaseError::Memory);
+    test_display_trait_impl!("Invalid ObjectHandle(ESYS_TR handle).", BaseError::BadTr);
+    test_display_trait_impl!(
+        "More than one session with decrypt set in SessionAttributes(TPMA_SESSION_DECRYPT bit set).",
         BaseError::MultipleDecryptSessions
     );
     test_display_trait_impl!(
-        "Multiple Encrypt Sessions",
+        "More than one session with encrypt set in SessionAttributes(TPMA_SESSION_ENCRYPT bit set).",
```

Comment:
I think a space before `(` is needed (otherwise I read it as function application :D ).